packages/contracts/package.json

```diff
@@ -11,13 +11,11 @@
         "build":
             "rm -rf ./lib; copyfiles ./build/**/* ./deploy/solc/solc_bin/* ./deploy/test/fixtures/contracts/**/* ./deploy/test/fixtures/contracts/* ./lib; tsc;",
         "test": "npm run build; truffle test",
-        "compile": "npm run build; node lib/deploy/cli.js compile",
+        "compile": "node ../deployer/lib/cli.js compile",
```

Comment:
There is an issue in yarn that makes it impossible to just use `"compile": "deployer compile",`
https://github.com/yarnpkg/yarn/issues/3846
because of that we can either refer to a top-level `node_modules` (`"compile": "node ../../node_modules/@0xproject/deployer/lib/cli.js compile",`) or leave it like that.
I'd prefer this option cause the upper one has the same issiue.
You can't refer to local node_modules cause yarn workspaces bubble tem up.

Reply:
Ok, then remove deployer from the devDependencies in `package.json` since this is an implicit dependency to whatever code happens to be in the `deployer` sub-package and the versions in `package.json` are ignored and therefore potentially confusing.

Reply:
I don't like that idea. That way at least lerna knows about the dependency and builds deployer before building contracts.